po/common/en.po

```diff
@@ -6503,8 +6503,8 @@ msgstr "Weight of one empty unit"
 
 # Unit = element, not unit of measure
 msgctxt "packaging_quantity_per_unit"
-msgid "Quantity of product contained per unit"
-msgstr "Quantity of product contained per unit"
+msgid "Net quantity of product contained per unit"
```

Comment:
```suggestion
msgid "Net quantity of product contained per unit (g)"
```
Doesn't it make sense to add the unit?

Reply:
it's not necessarily in g, it can be ml, it's like for portion, user should add the unit.

Reply:
Fine! Where's the explanation for all the users?